prow/sidecar/censor.go

```diff
@@ -93,6 +94,14 @@ func (o Options) censor() error {
 				return nil
 			}
 			logger := logrus.WithField("path", absPath)
+			relpath, _ := filepath.Rel(item, absPath) // err happens when there's no rel path, but we know there must be
```

Comment:
nit: If we're somehow wrong it is better to get a "Programmer error!" log rather than silence.